packages/location-field/src/utils.ts

```diff
@@ -21,10 +21,12 @@ const zipOrNeighborhood = ({
 // more layer types with custom rendering.
 const layerDisplayMap = {
   address: (properties: Properties): Label => {
-    const { locality, name, postalcode, region_a: state } = properties;
+    const { housenumber, locality, name, street, postalcode, region_a: state, region } = properties;
```

Comment:
Sort props

Reply:
Sorted in 0ee6e8c85ef16d0a68531803f17a22c393c8f5e2